lib/internal_metric/process.rb

```diff
@@ -13,7 +13,9 @@ class Process < Base
       rss: "Total RSS used by process",
       thread_count: "Total number of active threads per process",
       deferred_jobs_queued: "Number of jobs queued in the deferred job queue",
-      active_record_connections_count: "Total number of connections in ActiveRecord's connection pools"
+      active_record_connections_count: "Total number of connections in ActiveRecord's connection pools",
+      active_record_failover_count: "Count of activerecord backends in a failover state",
+      redis_failover_count: "Count of redis backends in a failover state"
```

Comment:
```suggestion
      redis_failover_count: "Count of Redis connections in a failover state"
```

Reply:
**Technically** it is not redis connections. There can be multiple connections to the same redis server. I think 'Redis servers' might be a better description?

```suggestion
      redis_failover_count: "Count of Redis servers in a failover state"
```